src/api/models/classification.py

```diff
@@ -0,0 +1,147 @@
+import uuid
+
+from django.conf import settings
+from django.contrib.gis.db import models
+from django.core.files.storage import FileSystemStorage
+from storages.backends.s3 import S3Storage
+
+from .base import BaseModel
+from .mermaid import BenthicAttribute, CollectRecord, GrowthForm, ObsBenthicPhotoQuadrat
+
+
+def select_image_storage():
+    if settings.DEBUG:
+        return FileSystemStorage()
+    else:
+        return S3Storage(bucket_name=settings.IMAGE_PROCESSING_BUCKET)
+
+
+class Label(BaseModel):
+    benthic_attribute = models.ForeignKey(
+        BenthicAttribute, related_name="labels", on_delete=models.CASCADE
+    )
+    growth_form = models.ForeignKey(GrowthForm, related_name="labels", on_delete=models.CASCADE)
```

Comment:
Just remembered that `growth_form` needs to be nullable -- most Labels do not in fact have a growth form.